crates/wkg/src/main.rs

```diff
@@ -371,21 +383,34 @@ impl GetArgs {
         } else {
             self.output
         };
-        ensure!(
-            self.overwrite || !output_path.exists(),
-            "{output_path:?} already exists; you can use '--overwrite' to overwrite it"
-        );
 
-        if let Some(wit) = wit {
-            std::fs::write(&output_path, wit)
-                .with_context(|| format!("Failed to write WIT to {output_path:?}"))?
+        if self.check {
+            let existing = std::fs::read(&output_path)
+                .with_context(|| format!("Failed to read {output_path:?}"))?;
+            let latest = if let Some(wit) = wit {
+                wit.into_bytes()
+            } else {
+                std::fs::read(&tmp_path).with_context(|| format!("Failed to read {tmp_path:?}"))?
```

Comment:
Nit: Should probably use `tokio` methods instead here since we're in async land. This shouldn't cause an issue with the CLI though, hence a nit

Reply:
Fixed